.env.deploy

```diff
@@ -18,13 +18,16 @@ MULTISITE_CLIENT_DOMAIN=""
 REDIS_HOST=""
 REDIS_PORT=""
 
+MAILGUN_DOMAIN=""
```

Comment:
These should just be `MAILGUN_DOMAIN=`
ie. don't provide a default value, because they *must* be configured